stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -93,10 +96,10 @@ class Process():
     def is_alive(self) -> bool: ...
     def join(self, timeout: Optional[float] = ...) -> None: ...
 
-class Queue():
+class Queue(queue.Queue[_T]):
     def __init__(self, maxsize: int = ...) -> None: ...
-    def get(self, block: bool = ..., timeout: float = ...) -> Any: ...
-    def put(self, item: Any, block: bool = ..., timeout: float = ...) -> None: ...
+    def get(self, block: bool = ..., timeout: Optional[float] = ...) -> Any: ...
+    def put(self, item: Any, block: bool = ..., timeout: Optional[float] = ...) -> None: ...
```

Comment:
The two `Any`s in `get()` and `put()` should be replaced with `_T`, to match the same methods in `queue.Queue`.